app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -33,7 +38,11 @@ keywords:
 expander:
   fees-and-funding:
     title: fees and funding
-    text: "Teacher training course fees for non-UK citizens cost an average of $fees_pgitt_internationalaverage$. Most non-UK citizens will not be eligible for tuition fee loans and maintenance loans. If you train to teach languages or physics, you may be eligible for a bursary or scholarship."
+    text: "Teacher training course fees for non-UK citizens cost an average of $fees_pgitt_internationalaverage$. Most non-UK citizens will not be eligible for tuition fee loans and maintenance loans. If you train to teach languages or physics, you may be eligible for a bursary or scholarship.
+    <p>If you have refugee status you can apply for salaried or fee-paying training courses. You will usually be eligible for financial support to help you train to teach.</p>
+    <p>You may be eligible for financial support to train to teach if <a href=\"/non-uk-teachers/visas-for-non-uk-trainees\"> you're from Ukraine or Afghanistan and on certain visas</a>.</p>
```

Comment:
```suggestion
    <p>You may be eligible for financial support to train to teach if <a href="/non-uk-teachers/visas-for-non-uk-trainees\">you're from Ukraine or Afghanistan and on certain visas</a>.</p>
```

Reply:
I added the extra slash to the link as it was the only way to make when I was doing this in VS Code. Shall remove it and see if it works now, but may need to change it back